packages/dd-trace/src/plugins/plugin.js

```diff
@@ -25,9 +25,31 @@ class Subscription {
   }
 }
 
+class StoreBinding {
+  constructor (event, transform) {
+    this._channel = dc.channel(event)
+    this._transform = data => {
+      const store = storage.getStore()
+
+      return !store || !store.noop
+        ? transform(data)
+        : store
```

Comment:
Wouldn't we want it to do the transform even if there _is_ a store so we can replace the active span store with the parent span store to escape in asyncStart bindStore calls?

Reply:
It already does, the condition is just a bit hard to read.

Reply:
Ah, right, read the `!store.noop` backwards. 😅